src/devices/Tca9548a/README.md

```diff
@@ -0,0 +1,101 @@
+            Console.WriteLine($"Calibration Status : {calibrationStatus}");
+            var magneto = bno055Sensor.Magnetometer;
+            Console.WriteLine($"Magnetometer X: {magneto.X} Y: {magneto.Y} Z: {magneto.Z}");
+            var gyro = bno055Sensor.Gyroscope;
+            Console.WriteLine($"Gyroscope X: {gyro.X} Y: {gyro.Y} Z: {gyro.Z}");
+            var accele = bno055Sensor.Accelerometer;
+            Console.WriteLine($"Acceleration X: {accele.X} Y: {accele.Y} Z: {accele.Z}");
+            var orien = bno055Sensor.Orientation;
+            Console.WriteLine($"Orientation Heading: {orien.X} Roll: {orien.Y} Pitch: {orien.Z}");
+            var line = bno055Sensor.LinearAcceleration;
+            Console.WriteLine($"Linear acceleration X: {line.X} Y: {line.Y} Z: {line.Z}");
+            var gravity = bno055Sensor.Gravity;
+            Console.WriteLine($"Gravity X: {gravity.X} Y: {gravity.Y} Z: {gravity.Z}");
+            var qua = bno055Sensor.Quaternion;
+            Console.WriteLine($"Quaternion X: {qua.X} Y: {qua.Y} Z: {qua.Z} W: {qua.W}");
+            var temp = bno055Sensor.Temperature.DegreesCelsius;
+            Console.WriteLine($"Temperature: {temp} °C");
+        }
+        else if (device == Bmp180.DefaultI2cAddress)
+        {
+            I2cConnectionSettings i2cSettings = new I2cConnectionSettings(i2cDevice.ConnectionSettings.BusId, Bmp180.DefaultI2cAddress);
+            using I2cDevice bmpDevice = I2cDevice.Create(i2cSettings);
+            using Bmp180 i2cBmp280 = new(bmpDevice);
+            i2cBmp280.SetSampling(Sampling.Standard);
+            Temperature tempValue = i2cBmp280.ReadTemperature();
+            Console.WriteLine($"Temperature: {tempValue.DegreesCelsius:0.#}\u00B0C");
+            Pressure preValue = i2cBmp280.ReadPressure();
+            Console.WriteLine($"Pressure: {preValue.Hectopascals:0.##}hPa");
+            Length altValue = i2cBmp280.ReadAltitude(WeatherHelper.MeanSeaLevel);
+            Console.WriteLine($"Altitude: {altValue:0.##}m");
+            Thread.Sleep(1000);
+            i2cBmp280.SetSampling(Sampling.UltraLowPower);
+            tempValue = i2cBmp280.ReadTemperature();
+            Console.WriteLine($"Temperature: {tempValue.DegreesCelsius:0.#}\u00B0C");
+            preValue = i2cBmp280.ReadPressure();
+            Console.WriteLine($"Pressure: {preValue.Hectopascals:0.##}hPa");
+            altValue = i2cBmp280.ReadAltitude(WeatherHelper.MeanSeaLevel);
+            Console.WriteLine($"Altitude: {altValue:0.##}m");
+        }
+        Thread.Sleep(1000);
+    }
+}
+```
+
+### Sample wiring
+
+**Important** to understand:
+If you are getting lower data rate from sensor connected via Multiplexer on Raspberry Pi, you need to adjust the bus speed.
+
+This example uses the I2C bus 1 with GPIO 2 and 3 on raspberry pi and gathers data from to BNO055 sensors and 4 BMP180 sensors over TCA9548A Mux
+
+![Wiring sample](TCA9548A_Connections.png)
```

Comment:
Thanks, that is much more helpful now. Do we still need the `TCA9548A-fritzing-schema.png` file?

Reply:
Will remove it.